README.md

```diff
@@ -15,6 +15,15 @@ Further bot  documentation [link](https://core.telegram.org/bots#6-botfather)
 2. Type `/newbot` then follow the instructions
 3. @botfather will send you your bot's authentication token
 
+#### Create a Github Access Token
+
+1. Goto your github account [settings](https://github.com/settings/profile)
+2. Click on [Developer Settings](https://github.com/settings/apps)
+3. Click on [Persoanl Access Tokens](https://github.com/settings/tokens)
+4. Here `Generate a New Token`.
+4. Give it a Name and all the Permissions and click on `Generate Token`.
```

Comment:
From looking at the code, I believe it might be that you'd only need `repo` permission instead of all permissions at this point. It would probably be safer to restrict the instructions to have folks enable just the permission they need.

Reply:
ok done